config/capsule-answers.yaml

```diff
@@ -0,0 +1,5 @@
+---
+certs:
+  generate: false
+  deploy: true
+capsule: true
```

Comment:
@mbacovsky, I lean towards having it default to 'true' since we always included it in the 'capsule-certs-generate' and there are bugs that exist when it is not enabled; however, I am open to other opinions.  @ehelms, thoughts?